packages/frontend/src/components/Files/CategoryFiles.tsx

```diff
@@ -124,6 +124,7 @@ export const CategoryFiles = ({ childFolderName, nestedFolderId }: TCategoryFile
           row={index}
           node={node}
           isSelected={isSelected(node)}
+          isStatic={node.entry.isFile || node.entry.isFolder}
```

Comment:
Any risk of `node` or `entry` being undefined?

Reply:
Just checked and shouldn't cause an issue here